chance.js

```diff
@@ -1272,6 +1272,13 @@
         return nationality.name;
     };
 
+     // Generate random zodiac sign
+     Chance.prototype.zodiac = function () {
+        const zodiacSymbols = ["Aries","Taurus","Gemini","Cancer","Leo","Virgo","Libra","Scorpio","Sagittarius","Capricorn","Aquarius","Pisces"];
+        return zodiacSymbols[Math.floor(Math.random() * zodiacSymbols.length)];
```

Comment:
Also please do not write your own selector like this using `Math.floor()` and `Math.random()` and whatnot, please just use `chance.pickOne`, [for example ](https://github.com/chancejs/chancejs/blob/d292eeaee07240e18a333664462d216bf3921051/chance.js#L2187)

```suggestion
        return this.pickone(zodiacSymbols);
```

By taking the approach you are, you are circumventing all of the repeatability and seed functionality that the in-built Chance random number generator provides.

Sorry I didn't catch this on my first review here!

Reply:
🤯

Reply:
Sorry, It was my fault, I didn't see the method.